### ISSUE TITLE
All navigation history is lost, until you open the Navigator side bar

### ISSUE THREAD

**Issue by Zyrsha** [dakaraphi/vscode-extension-navigator#2]
I tried to use this extension, but whenever I wished to see my history, I found that is was empty. The reason it is empty is because it does not start recording any history until you open up the side bar.

Can this be fixed so that your navigation history is still kept, even if you haven't opened the side bar yet?

**Comment by dakaraphi** [comment]
@Zyrsha yes, this was already something I've planned to fix, unfortunately other work has kept me very busy.  As soon as I can get back to this, it will be the first thing I fix.
